server/ws/server.py

```
@@ -47,6 +47,8 @@ def check_connections(logger):
 
 
 async def _forward_messages(logger):
+    _PING_INTERVAL = 5  # 秒（原為每 0.5 秒，改為 5 秒以減少不必要的開銷）
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)